traefik/values.yaml

```diff
@@ -8,7 +8,6 @@ image:
 # Configure the deployment
 #
 deployment:
-  enabled: true
```

Comment:
Regarding this PR https://github.com/containous/traefik-helm-chart/pull/110 and this issue https://github.com/containous/traefik-helm-chart/issues/68, could you please keep the `enable` field.

Reply:
I agree. `enabled` field should be kept.

Reply:
The entire premise of #68 is based on a lack of proper helm usage and hook definition. Helm will install CRDs prior to the rest and IngressRoutes only after set properly _even when using helm template_. I strongly disagree with this flag existing at all.

Reply:
Ill leave it for now because there are so many things that need to be fixed but i would like to circle back to this later